content/_en.html

```diff
@@ -3,75 +3,347 @@
 linkTitle: "CUE"
 ---
 
-<section class="section section--blue section--center">
+<section class="section section--blue section--banner">
     <header class="section__header">
         <div class="section__content">
             <div class="banner">
                 <div class="banner__content">
-                    <h1 class="banner__title">alpha.cuelang.org is currently under active development</h1>
-                    <p>We welcome feedback and issues via the <a
-                        href="https://github.com/cue-lang/cue">CUE issue
-                        tracker</a>.</p>
+                    <h1 class="banner__title">Configure Unify Execute</h1>
+                    <p>Validate, define, and use dynamic and text-based data</p>
```

Comment:
maybe us a non-breakable-hyphen here so the word 'text-based' stays together. see https://stackoverflow.com/questions/8753296/how-to-prevent-line-break-at-hyphens-in-all-browsers